inspirehep/modules/records/jsonschemas/records/authors.json

```diff
@@ -105,13 +104,13 @@
                             "curated_relation": {
                                 "type": "boolean"
                             },
-                            "recid": {
-                                "type": "integer",
-                                "description": "Corresponding record ID in the institution database"
-                            },
                             "name": {
                                 "type": "string",
                                 "description": "The raw institution name"
+                            },
+                            "record": {
+                                "description": "Corresponding URI for the institution record",
```

Comment:
The schema validator didn't complain about this. Also description and title are informative and don't tell anything about the structure of the object.